### DISCUSSION TITLE
You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.

### DISCUSSION THREAD

**Discussion by WordpressGithub** [community/community#43148] [GitHub Education]
I am significantly getting error message showing:
**Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.**
Please help
I have updated my profile. Changed Profile Image.
Tried Many times. Uploaded School ID card, Syllabus, Website images.
Still not working....

**Comment by LuisContrerasGlz** [comment]
If you have already tried these things and your application is still not being reviewed, you may want to try reaching out to the GitHub team for assistance. You can contact them through the "Contact Support" link in the footer of the GitHub website.

**Comment by sriram-gajula** [comment]
I'm also getting same problem. Can any one please help me out

**Comment by jfullstackdev** [comment]
posting it here might help, 

contacting Support might help too

https://support.github.com/contact/account
